README.md

```diff
@@ -20,92 +20,87 @@
 
 # What is Apache Submarine?
 
-Apache Submarine is a unified AI platform which allows engineers and data scientists to run Machine Learning and Deep Learning workload in distributed cluster.
+Apache Submarine (Submarine for short) is the `ONE PLATFORM` to allow Data Scientists to create end-to-end machine learning workflow. `ONE PLATFORM` means it supports Data Scientists and data engineers to finish their jobs on the same platform without frequently switching their toolsets. From dataset exploring data pipeline creation, model training (experiments), and push model to production (model serving and monitoring). All these steps can be completed within the `ONE PLATFORM`.
```

Comment:
What features are we going to support for data engineers?

Reply:
change `data engineers`  to `Data Engineers`?

Reply:
It may make sense to focus on "Data Scientist" use cases for now. Adding things we won't support will dilute our messaging. I just updated the doc.